### ISSUE TITLE
How to create relation between modules

### ISSUE THREAD

**Issue by te-deum** [YetiForceCompany/YetiForceCRM#7812]
Hello,

#### Issue
I am working on YetiForce 4.4.0 (Fresh installation) and I create some modules (with the Module Manager). I can create 1:M relation field with Layout Editor but I don't know how to add other relation (1:1, M:M) and/or bi-directional relations.

Is there any tips to create relations between modules ?

Thank you for your support.

#### Your Environment

* YetiForce Version used: 4.4.0
* Browser name and version: Firefox / Chrome
* Environment name and version: Windows
* Operating System and version: CentOS Linux 7.5.1804

**Comment by vovpff** [comment]
@te-deum enable this option and you get Add Relation button https://github.com/YetiForceCompany/YetiForceCRM/blob/c4d8a3fd03a34b06947faa0011b1a7524561bfad/config/developer.php#L18

**Comment by ldgbc** [comment]
To follow up on vovpff, after that you can go to here:
/index.php?module=LayoutEditor&parent=Settings&view=Index&mode=showRelatedListLayout&block=2&fieldid=41

And add your relation from Source (e.g. Contact) to Target (e.g. te-duem MODULE).
With regard to the Many to Many and Many to Many with Separation of data. I'm not sure what the differences is yet. See issue: https://github.com/YetiForceCompany/YetiForceCRM/issues/7482

**Comment by te-deum** [comment]
Thank you for your response.
I can now add my relations !

**Comment by Pietrox** [comment]
I just did that, but there is no "add relation button" - am I missing something? Something with cron, sessions or templates_c folder to delete maybe?

**Comment by vovpff** [comment]
You need to activate this feature in /config/relation.php

**Comment by Pietrox** [comment]
Checked and seems fine but I had to restart apache2 and after reload it popped up.
So all is fine. But I do have one question.
If I have Accounts module and custom "Transactions" module where in Accounts I have custom userID which is unique and in Transactions I have also in multiple records userID which are supposed to be assigned to an account. 
How do I make it to automatically assign Transactions to Accounts based on matching field? Just like tickets and mailscanner by email.
